rosidl_parser/test/test_parser.py

```diff
@@ -315,13 +315,13 @@ def test_service_parser(service_idl_file):
     assert srv.namespaced_type.name == 'MyService'
     assert len(srv.request_message.structure.members) == 2
     assert len(srv.response_message.structure.members) == 1
-    assert(srv.event_message.structure.namespaced_type.name ==
+    assert (srv.event_message.structure.namespaced_type.name ==
            'MyService' + SERVICE_EVENT_MESSAGE_SUFFIX)
```

Comment:
Are the parentheses here required because of the line break?

```suggestion
    assert srv.event_message.structure.namespaced_type.name ==
           'MyService' + SERVICE_EVENT_MESSAGE_SUFFIX
```

Reply:
Yeah, exactly.  Without the parentheses, Python actually thinks this is invalid syntax.  We might also be able to fix it with a `\` at the end of the line, but I thought the parentheses was clearer in this case.